lib/guild/controller.js

```diff
@@ -37,6 +57,8 @@ function members(req, res, app) {
         });
 
         promises.push(promise);
+        // delay for 20 Milliseconds..
+        sleep(100);
```

Comment:
http://www.w3schools.com/js/js_timing.asp  You could just wrap the entire promise creation and push into setTimeout.  No biggy right now though as this stuff will end up getting a slight refactor when we turn it into a service.